<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleSetStationPage</class>
 <widget class="QWidget" name="VehicleSetStationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 50);</string>
  </property>
  <widget class="QPushButton" name="BTN7">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN3">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNHome">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>620</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/Home.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN4">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN8">
   <property name="geometry">
    <rect>
     <x>724</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN9">
   <property name="geometry">
    <rect>
     <x>826</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN5">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN2">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>34</y>
     <width>1024</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 142, 68);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_line">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>40</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 142, 68);
font:24px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>线路1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNOriginStation">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px &quot;微软雅黑&quot;;
border-radius:8px;
background-color: rgb(233, 146, 68);</string>
   </property>
   <property name="text">
    <string>起始站</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_OriginStation">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>46</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-right: 2px solid rgb(255,188,124);
border-bottom:2px solid rgb(255,188,124);
font:24px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Terminal">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>44</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-right: 2px solid rgb(255,188,124);
border-bottom:2px solid rgb(255,188,124);
font:24px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTerminal">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>40</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px &quot;微软雅黑&quot;;
border-radius:8px;
background-color: rgb(233, 146, 68);</string>
   </property>
   <property name="text">
    <string>终点站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>西二环路</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>孔家营</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>3</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>呼钢东路</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation4">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>160</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>4</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>西龙王庙</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>7</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>新华广场</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>6</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>附属医院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>5</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>马兰夫纪念馆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation8">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>240</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>8</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>博物馆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation11">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>11</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>东影路</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation10">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>10</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>艺术学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>9</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>将军衙署</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation12">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>330</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>12</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>内蒙古展览馆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation15">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>420</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>15</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>呼和浩特东站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation14">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>14</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>市政府</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>13</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>乌兰恰特</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation16">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>420</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>16</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>后不塔气</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation19">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>510</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>19</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>堰坝村</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation18">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>18</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>白塔西</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStation17">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>17</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>什兰岱村</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNChangeOrigin_Terminal">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>46</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/arrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
